web/template/payments.gohtml

```diff
@@ -15,9 +15,47 @@
             {{ else }}
                 <table class="govuk-table my-table table__no-border">
                     <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        {{ if eq .OutstandingFee 0 }}
+                            <th scope="row" class="govuk-table__header">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">No further fees due</h2>
+                            </th>
+                            <td class="govuk-table__cell">{{/*required for table styling*/}}</td>
```

Comment:
Added a blank table cell to apply the border styling.. couldn't think of a better solution...
<img width="425" alt="Screenshot 2022-10-05 at 09 28 30" src="https://user-images.githubusercontent.com/51047911/194029208-5a3cf843-993b-48a3-ab58-67e442d768f0.png">